### DISCUSSION TITLE
Komodo

### DISCUSSION THREAD

**Discussion by dudel** [selfhst/icons#165] [New Icon Request]
Github: https://github.com/mbecker20/komodo
Homepage: https://komo.do/

Thanks!

**Comment by shollyethan** [comment]
The current Komodo icon in the repo isn't great quality. I'm going to reach out to the devs to see if I can get something better - will keep this discussion updated when I hear back.

**Comment by shollyethan** [comment]
> Github: https://github.com/mbecker20/komodo Homepage: https://komo.do/
> 
> Thanks!

Hi - the Komodo devs were graciously able to supply an SVG icon that has officially been added to the collection. Apologies for the delay!
